src/NpmUtilities.js

```diff
@@ -157,11 +157,9 @@ export default class NpmUtilities {
       cwd: directory,
     };
 
-    if (registry) {
-      opts.env = Object.assign({}, process.env, {
-        npm_config_registry: registry,
-      });
-    }
+    opts.env = Object.assign({}, process.env, {
+      npm_config_registry: registry || 'https://registry.npmjs.org/',
```

Comment:
This should default to the fully-qualified URL: `"https://registry.npmjs.org/"`

Reply:
then what if the user specifies a custom registry? `https://registry.npmjs.org/` is default if there's no registry set iirc

Reply:
Oh, then... right now they’ll have to specify that in the lerna options. This is one of the drawbacks to _always_ passing the registry env var.

Reply:
ok updated